frontend/src/util/auth.tsx

```diff
@@ -15,6 +17,9 @@ interface User {
 
 const fakeFirebaseUser: User = {
 	async getIdToken(): Promise<string> {
+		if (AUTH_MODE === 'password') {
+			return sessionStorage.getItem('passwordToken') || ''
+		}
```

Comment:
clever! this allows `graph.tsx` to use this method to send the token, took me a sec to realize that